### DISCUSSION TITLE
multiple grib records

### DISCUSSION THREAD

**Discussion by Deba57** [dtcenter/METplus#1109] [File I/O] [answered]
Hi,
I am trying to read a grib record from a file where multiple records of "PMTF" are present. Wgrib2 -V of the file is attached here. How in MET I specify particular record, in here would like to read number 111. Any help is appreciated.
[testconc2.txt](https://github.com/dtcenter/METplus/files/7042837/testconc2.txt)

**Comment by malloryprow** [comment, marked as answer]
For the "level" entry for your variable information you can use "R111". 

From the MET documentation (https://met.readthedocs.io/en/latest/Users_Guide/config_options.html): "The “level” entry specifies a level type and value: RNNN for a specific GRIB record number"
